addon/types/model_export/model.py

```diff
@@ -172,13 +177,36 @@ def generate_qc(self):
             qc.write('$mostlyopaque')
             qc.write('\n')
 
+        output_x = 0
+        output_y = 0
+        output_z = 0
+        output_rotation = 0
+        output_scale = 1
+
+        if self.custom_transform_source == 'MANUAL':
+            output_x = self.origin_x
+            output_y = self.origin_y
+            output_z = self.origin_z
+            output_rotation = self.rotation
+            output_scale = self.scale
+        elif self.custom_transform_source == 'CUSTOM_OBJECT' and self.custom_transform_object_ref is not None:
+            output_x = (self.custom_transform_object_ref.matrix_world[0][3]) * -1
+            output_y = self.custom_transform_object_ref.matrix_world[1][3]
+            output_z = (self.custom_transform_object_ref.matrix_world[2][3]) * -1
+            output_rotation = degrees(self.custom_transform_object_ref.rotation_euler.z)
```

Comment:
Rotation can be extracted from the matrix; do that instead of taking it from the object directly.